### PR TITLE
Update the language option file to database

### PR DESCRIPTION
**Pull request by mohamedlafiteh**

### CODE REVIEW

File: client/src/components/LanguageOptions.js [hunk 1]
```diff
@@ -1,25 +1,13 @@
...
-  { key: "Viet", text: "Vietnamese", value: "Vietnamese" }
+  { key: "zh-cn", text: " MandarinChinese", value: "Mandarin Chinese" },
+  { key: "ar", text: "Arabic", value: "Arabic" },
+  { key: "es", text: "Spanish", value: "Spanish" },
+  { key: "ml", text: "Danish", value: "Malaysian" },
+
+  { key: "en", text: "English", value: "English" },
+  { key: "ru", text: "Russian", value: "Russian" },
+  { key: "be", text: "Russian", value: "Bengali" },
+  { key: "Fr", text: "French", value: "French" }
```
Comment (studio-sherpa) [inline client/src/components/LanguageOptions.js]: change the key Fr to lower case: fr
Every language code should be in lower case

File: client/src/components/LanguageOptions.js [hunk 2]
```diff
@@ -1,25 +1,13 @@
...
-  { key: "Lith", text: "Lithuanian", value: "Lithuanian" },
-  { key: "Per", text: "Persian", value: "Persian" },
-  { key: "Pol", text: "Polish", value: "Polish" },
-  { key: "Por", text: "Portuguese", value: "Portuguese" },
-  { key: "Rus", text: "Russian", value: "Russian" },
-  { key: "Spa", text: "Spanish", value: "Spanish" },
-  { key: "Swe", text: "Swedish", value: "Swedish" },
-  { key: "Tur", text: "Turkish", value: "Turkish" },
-  { key: "Viet", text: "Vietnamese", value: "Vietnamese" }
+  { key: "zh-cn", text: " MandarinChinese", value: "Mandarin Chinese" },
```
Comment (studio-sherpa) [inline client/src/components/LanguageOptions.js]: You don't need the text in every language, with the key and the value is enough

File: client/src/components/LanguageOptions.js (hunk 1 above)
Comment (mohamedlafiteh) [reply client/src/components/LanguageOptions.js]: I changed it, in the database only have code and name, but in the options we have key, text and value, should i change it?
when i was testing it on the frondend is working but

File: client/src/components/LanguageOptions.js (hunk 2 above)
Comment (mohamedlafiteh) [reply client/src/components/LanguageOptions.js]: it is not working when i remove the text in the front end

Comment (studio-sherpa) [reply client/src/components/LanguageOptions.js]: you are right, we need text and value. But the value should contain what we call the 'language code' eg: fr --> for French,  es --> for spanish

File: client/src/components/LanguageOptions.js [hunk 3]
```diff
@@ -1,13 +1,13 @@
 const languageOptions = [
-  { key: "zh-cn", text: " MandarinChinese", value: "Mandarin Chinese" },
-  { key: "ar", text: "Arabic", value: "Arabic" },
-  { key: "es", text: "Spanish", value: "Spanish" },
-  { key: "ml", text: "Danish", value: "Malaysian" },
+  { text: " MandarinChinese", value: "zh-cn" },
+  { text: "Arabic", value: "ar" },
+  { text: "Spanish", value: "es" },
+  { text: "Danish", value: "ml" },
```
Comment (studio-sherpa) [inline client/src/components/LanguageOptions.js]: the text should be "Malaysian"

File: client/src/components/LanguageOptions.js [hunk 4]
```diff
@@ -1,13 +1,13 @@
...
 
-  { key: "en", text: "English", value: "English" },
-  { key: "ru", text: "Russian", value: "Russian" },
-  { key: "be", text: "Russian", value: "Bengali" },
-  { key: "Fr", text: "French", value: "French" }
+  { text: "English", value: "en" },
+  { text: "Russian", value: "ru" },
+  { text: "Russian", value: "be" },
```
Comment (studio-sherpa) [inline client/src/components/LanguageOptions.js]: the text should be Bengali

Comment (mohamedlafiteh) [reply client/src/components/LanguageOptions.js]: I changed it

File: client/src/components/LanguageOptions.js (hunk 3 above)
Comment (mohamedlafiteh) [reply client/src/components/LanguageOptions.js]: i changed it

**Review by NadaKartouch — APPROVED** [review]

**Review by NadaKartouch — APPROVED** [review]

File: client/src/components/LanguageOptions.js [hunk 5]
```diff
@@ -1,25 +1,13 @@
...
-  { key: "Lith", text: "Lithuanian", value: "Lithuanian" },
-  { key: "Per", text: "Persian", value: "Persian" },
-  { key: "Pol", text: "Polish", value: "Polish" },
-  { key: "Por", text: "Portuguese", value: "Portuguese" },
-  { key: "Rus", text: "Russian", value: "Russian" },
-  { key: "Spa", text: "Spanish", value: "Spanish" },
-  { key: "Swe", text: "Swedish", value: "Swedish" },
-  { key: "Tur", text: "Turkish", value: "Turkish" },
-  { key: "Viet", text: "Vietnamese", value: "Vietnamese" }
+  { text: " MandarinChinese", value: "zh-cn" },
```
Comment (llh1) [inline client/src/components/LanguageOptions.js]: The text should be "Mandarin Chinese".

**Review by llh1 — CHANGES_REQUESTED** [review]

File: client/src/components/LanguageOptions.js (hunk 5 above)
Comment (mohamedlafiteh) [reply client/src/components/LanguageOptions.js]: it is fixed

File: client/src/components/LanguageOptions.js [hunk 6]
```diff
@@ -1,25 +1,13 @@
...
-  { key: "Lith", text: "Lithuanian", value: "Lithuanian" },
-  { key: "Per", text: "Persian", value: "Persian" },
-  { key: "Pol", text: "Polish", value: "Polish" },
-  { key: "Por", text: "Portuguese", value: "Portuguese" },
-  { key: "Rus", text: "Russian", value: "Russian" },
-  { key: "Spa", text: "Spanish", value: "Spanish" },
-  { key: "Swe", text: "Swedish", value: "Swedish" },
-  { key: "Tur", text: "Turkish", value: "Turkish" },
-  { key: "Viet", text: "Vietnamese", value: "Vietnamese" }
+  { text: " Mandarin Chinese", value: "zh-cn" },
```
Comment (llh1) [inline client/src/components/LanguageOptions.js]: Still a problem: remove the space at the beginning :)

**Review by llh1 — CHANGES_REQUESTED** [review]

File: client/src/components/LanguageOptions.js (hunk 6 above)
Comment (mohamedlafiteh) [reply client/src/components/LanguageOptions.js]: I removed the space

**Review by llh1 — APPROVED** [review]

**Pull request merged**